reviews/views.py

```diff
@@ -1,3 +1,62 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http      import JsonResponse, request
+from django.views     import View
+from reviews.models   import Review
+from users.models     import User
+from products.models  import Product
+from users.utils      import login_decorator
+
+class ReviewView(View):
+    def get(self,request, review_id):
+        reviews = Review.objects.all()
+        result = []
+
+        if not Review.objects.filter(id=review_id).exists():
+            return JsonResponse ({"MESSAGE": "REVIEW_DOES_NOT_EXISTS"}, status = 400)   
+ 
+        result = [
+            {
+                "user"            : review.user.id,
+                "product_id"      : review.product_id ,
+                "create_at"       : review.created_at,
+                "size_rating"     : review.size_rating,
+                "color_rating"    : review.color_rating,
+                "delivery_rating" : review.delivery_rating,
+                "title"           : review.title,
+                "comment"         : review.comment,
+
+            } for review in reviews]
+
+        return JsonResponse ({"result": result}, status = 200)
+
+    @login_decorator
+    def delete (self,request,review_id):
+
+        if not Review.objects.filter(id=review_id).exists():
+            return JsonResponse({"MESSAGE": "NO_REVIEWS"}, status = 400)
+
+        if Review.objects.get(id=review_id) != request.user :
+            return JsonResponse({"MESSAGE": "UNAUTHROIZED"}, status = 400)
+
+        Review.objects.filter(id=review_id).delete()
```

Comment:
```suggestion
        if not Review.objects.filter(id=review_id, user=request.user).exists():
            return JsonResponse({"MESSAGE": "NO_REVIEWS"}, status = 400)

        Review.objects.filter(id=review_id, request.user).delete()
```
patch에 남긴 리뷰와 동일한 이유 입니다.